public/lessons/client_side_coding/media.md

```diff
@@ -1,32 +1,54 @@
 
-JPGs and JPEGs are the same type of image, however when embedding them in your page you will need to make sure you are using the correct file extension otherwise your site won't be able to locate the image
+JPGs and JPEGs are the same type of image, however when embedding them in your page you will need to make sure you are using the correct file extension otherwise your site won't be able to locate the image. Also **case matters!**
 
-### png
+# png
 
-PNGs, like JPGs, are a pixel based image type, and as such do not scale well beyond their original size. However, unlike JPGs, PNGs suppport transparency, and as such can appear to be irregular shapes.
+PNGs, like JPGs, are a pixel based image type, and as such do not scale well beyond their original size. However, unlike JPGs, PNGs support transparency, and as such can appear to be irregular shapes.
 
-Despite how they appeare they are still rectangular and as such will prevent you from interacting with elements that are behind them even though you can see through them.
+Despite how they appear they are still rectangular and as such will prevent you from interacting with elements that are behind them even though you can see through them.
 
-### svg
+# svg
 
 SVG stands for **standard vector graphic** and unlike other types of images they are not made out of pixels.  They are instead collections of instructoins that tell the computer how to draw a specific shape.
 
-Since they are not blocks of pixels SVGs can be scaled up indefinately with no detrinmental effects on quality. They also are **exactly** the shape they appear to be and as such will only block what they appear to cover.
+Since they are not blocks of pixels SVGs can be scaled up indefinately with no detrinmental effects on quality, and they have very small file sizes compared to the other two image types covered above. They also are **exactly** the shape they appear to be and as such will only block what they appear to cover.
+
+# Lab: Embedding Images
+
+Let's go to our trusty "example-html" directory, and add some images to make your page more engaging!
+
+* add in one image from an external source per section on your `index.html` document
+  * There are many sites around the web that serve up placeholder images. (PlaceKitten)[https://placekitten.com/] is the classic, though (PlaceCage)[https://www.placecage.com/], and (FillMurray)[https://www.fillmurray.com/] are both hilarious
+
+* Choose an image from your computer to include on your About page
+* Create a subdirectory named "images" (or 'imgs', or 'pictures', or whatever name makes sense to you)
+* Place a copy of your chosen image into the newly created subdirectory.
+* Embed that image into your About page.
+* Bonus Challenge: What happens when you wrap an image in an anchor tag?
+
 
 # Video
 
```

Comment:
conditions* on line 61, it wouldn't let me put the comment on the line since it wasn't changed